.typos.toml

```diff
@@ -11,6 +11,7 @@
 
 # Match Inside a Word - Case Insensitive
 [default.extend-words]
+ACI = "ACI"
```

Comment:
This should go in identifiers with both all lowercase and all uppercase, otherwise any incorrectly spelled word that happens to contain "aci" wouldn't get picked up.

Reply:
I see..

Reply:
I guess it actually needed `aci_palette`. Run `typos` locally! :)